extension/src/popup/components/manageAssets/SearchAsset/index.tsx

```diff
@@ -0,0 +1,194 @@
+import React, { useEffect, useCallback, useRef, useState } from "react";
+import { useSelector } from "react-redux";
+import { Link } from "react-router-dom";
+import { Formik, Form, Field, FieldProps } from "formik";
+import { Input, Loader } from "@stellar/design-system";
+import debounce from "lodash/debounce";
+
+import { Button } from "popup/basics/buttons/Button";
+import { InfoBlock } from "popup/basics/InfoBlock";
+import { FormRows } from "popup/basics/Forms";
+
+import { ROUTES } from "popup/constants/routes";
+
+import { settingsNetworkDetailsSelector } from "popup/ducks/settings";
+
+import { SubviewHeader } from "popup/components/SubviewHeader";
+
+import { ManageAssetRows, ManageAssetCurrency } from "../ManageAssetRows";
+
+import "./styles.scss";
+
+interface FormValues {
+  asset: string;
+}
+const initialValues: FormValues = {
+  asset: "",
+};
+
+interface SearchAssetProps {
+  setErrorAsset: (errorAsset: string) => void;
+}
+
+const AddManualAssetLink = () => (
+  <div className="SearchAsset__copy">
+    Can’t find the asset you’re looking for?
+    <div>
+      <Link to={ROUTES.addAsset}>Add it manually</Link>
+    </div>
+  </div>
+);
+
+export const SearchAsset = ({ setErrorAsset }: SearchAssetProps) => {
+  const { isTestnet } = useSelector(settingsNetworkDetailsSelector);
+  const [assetRows, setAssetRows] = useState([] as ManageAssetCurrency[]);
+  const [maxHeight, setMaxHeight] = useState(0);
+  const [isSearching, setIsSearching] = useState(false);
+  const [hasNoResults, setHasNoResults] = useState(false);
+  const ResultsRef = useRef<HTMLDivElement>(null);
+
+  const ResultsHeader = () => (
```

Comment:
nit - any reason `ResultsHeader` needs to be defined w/in SearchAsset? Could we define outside of SearchAsset to be consistent with the other sub-component `AddManualAssetLink` (or both defined w/in, I think outside reads easier though)

Reply:
Ah yes, good catch!